src/main/java/org/jenkinsci/plugins/configfiles/buildwrapper/ManagedFile.java

```diff
@@ -108,8 +110,10 @@ public String getDisplayName() {
 
         public ListBoxModel doFillFileIdItems(@AncestorInPath ItemGroup context, @AncestorInPath Item project) {
             // You should have permission to configure your project in order to get the available managed files
-            if (project != null) {
+            if (context != null) {
                 project.checkPermission(Item.CONFIGURE);
+            } else {
+                Jenkins.get().checkPermission(Item.CREATE);
             }
```

Comment:
I think that @jtnord is correct that we should be checking project for null rather than context.
```suggestion
            if (project != null) {
                project.checkPermission(Item.CONFIGURE);
            } else {
                Jenkins.get().checkPermission(Item.CREATE);
            }
```